### PR TITLE
Lint Ruff 0.8.0 and bump imgui-bundle version to match wgpu-py

### PR DESCRIPTION
**Pull request by claydugo**
Blocked CI in https://github.com/pygfx/pygfx/pull/876

### CODE REVIEW

**Review by almarklein — APPROVED** [review]

**Comment by almarklein** [comment]
@korijn, let's bypass the checks and merge this one? Otherwise we need both issues to be resolves in a single pr 😅

**Review by Korijn — DISMISSED** [review]

**Comment by claydugo** [comment]
I moved the [imgui version bump PR](https://github.com/pygfx/pygfx/pull/891) into this one to simplify it a bit.

The rest of the failures should pass after the next `wgpu-py` release.

**Review by Korijn — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ examples = [
     "scikit-image",
     "trimesh",
     "gltflib",
-    "imgui-bundle>=1.2.1",
+    "imgui-bundle>=1.6.0",
```
Comment (hmaarrfk) [inline pyproject.toml:37]: this can be reverted right? i feel like we loosened the requirements on wgpu-py

Comment (hmaarrfk) [reply pyproject.toml:37]: i guess its for the examples only, and users won't get pip complaining so its probably a no-op.

**Pull request merged**